mRemoteV1/Connection/Protocol/Http/Connection.Protocol.HTTPBase.cs

```diff
@@ -68,12 +70,13 @@ public override bool Initialize()
 				if (InterfaceControl.Info.RenderingEngine == RenderingEngine.Gecko)
 				{
 				    var GeckoBrowser = (GeckoWebBrowser) wBrowser;
-                    if (GeckoBrowser != null)
-                    {
-                        GeckoBrowser.DocumentTitleChanged += geckoBrowser_DocumentTitleChanged;
-                        //GeckoBrowser.Tab.LastTabRemoved += wBrowser_LastTabRemoved;
-                    }
-                }
+				    if (GeckoBrowser == null) return false;
+				    GeckoBrowser.DocumentTitleChanged += geckoBrowser_DocumentTitleChanged;
+				    //GeckoBrowser.Tab.LastTabRemoved += wBrowser_LastTabRemoved;
+
+                    if(InterfaceControl.Info.HttpsTrustInsecureCerts)
```

Comment:
This check might be unnecessary since we're doing it later... But I suppose there is no need for the event handler if we know we're not going to do anything with it.